src/Avalonia.Controls/Slider.cs

```diff
@@ -94,6 +102,15 @@ public Slider()
             UpdatePseudoClasses(Orientation);
         }
 
+        /// <summary>
+        /// Defines the ticks to be drawn on the tick bar.
+        /// </summary>
+        public List<double> Ticks
```

Comment:
I think making this a `AvaloniaList<double>` allows defining this in XAML `Ticks="0, 5, 20, 100"`

Reply:
ah true... it should be avalonia list

Reply:
The [Ticks property in TickBar](https://github.com/AvaloniaUI/Avalonia/blob/535c053338cea3177f1b7ba4b8ed1863516ba07d/src/Avalonia.Controls/TickBar.cs#L138) is also a `List<double>`, therefore we should change this first maybe.

Reply:
@Mehigh17 perhaps you can change both in this PR?

Reply:
@jmacato Fixed in [last commit](https://github.com/AvaloniaUI/Avalonia/pull/4219/commits/d8d9a1865dc4abe7466a43f14edbab7262222f7a).